CODE/Makefile

```diff
@@ -1,4 +1,9 @@
+##intel fortran
 F90=mpiifort
+
```

Comment:
nice, what about passing the option of type of compiler through as argument 
i.e. intel, gcc etc 
see here
https://stackoverflow.com/a/2826178/4679900

Reply:
i guess eventually would be nice to have a separate config step and then we do make all or something 😄